drivers/network/tcpip/lwip/.gitignore

```diff
@@ -0,0 +1,60 @@
+*.o
+*.so
+*.a
+*.exe
+*lwip_unittests.xml
+*.suo
+*.log
+.depend*
+
+/.vscode/ipch
+
+/doc/doxygen/output/html
+
+/test/fuzz/output
+/test/fuzz/lwip_fuzz
+/test/fuzz/lwip_fuzz2
+/test/fuzz/lwip_fuzz3
+/test/fuzz/.depend
+
+/build
+
+/contrib/examples/example_app/lwipcfg.h
+/contrib/examples/example_app/lwipopts_test.h
+
+/contrib/ports/unix/example_app/build
+/contrib/ports/unix/example_app/example_app
+/contrib/ports/unix/example_app/makefsdata
+
+/contrib/ports/unix/check/build
+/contrib/ports/unix/lib/build
+/contrib/ports/unix/check/lwip_unittests
+
+/contrib/ports/win32/example_app/build
+/contrib/ports/win32/lwipcfg_msvc.h
+
```

Comment:
Can't we just get rid of all that crap that we won't ever need at all? This adds a ton of stuff in our source tree (and in the PR) that just adds weight for nothing.

Reply:
Removed all contrib directory